Docker/base-images/entando-dependencies-base/settings.xml

```diff
@@ -9,7 +9,7 @@
             <id>mirror.default</id>
             <mirrorOf>nothing</mirrorOf>
             <name>An internal openshift mirror</name>
-            <url>http://172.17.0.3:8081/repository/maven-public/</url>
+            <url>http://192.168.0.107:8081/repository/maven-public/</url>
```

Comment:
Why did this IP move?

Reply:
This mirror is inactive by default (<mirrorOf>nothing</mirrorOf) and the idea is just for whoever is maintaining the images to be able to build and test fully locally. During intense development and testing I often run a build every 5-10 minutes, and having a local Nexus speeds up the process significantly but also saves you some bandwdith. The IP address could therefore be any known Nexus instance, wherever the image maintainer has decided to make one available. The 172.17.* address was an internal Docker ip, but I had to change it to my machine's external IP (192.168.*) to test Docker images that ran in Openshift only because they had no awareness of my local Docker network.